vite.config.js

```diff
@@ -1,55 +1,63 @@
-import { defineConfig } from 'vite'
+import { defineConfig, loadEnv } from 'vite'
 import react from '@vitejs/plugin-react'
 import path from 'path';
-import getEnvModule from './env'
 import dns from 'dns'
 
-dns.setDefaultResultOrder('verbatim')
+// https://vitejs.dev/config/
+export default defineConfig(({ _, mode }) => {
 
-getEnvModule().setEnvironmentVarsFromDir(__dirname);
+  const env = loadEnv(mode, process.cwd(), '')
```

Comment:
Could we do this the way it is done in `main` and load `.okta.env` instead of `.env` so that there is less drift from upstream?

```
import dotenv from 'dotenv'
// ...
dotenv.config({path: '.okta.env'})
```

Reply:
I put the statement back to load the `.okta.env` if available.